Gemfile

```diff
@@ -26,7 +26,7 @@ group :development, :test do
   gem "faker"
   gem "i18n-tasks"
   gem "pry-rails"
-  gem "rspec-rails", "~> 3.4.2"
+  gem "fuubar", "~> 2.1.0.beta2"
```

Comment:
It's not removed, it's a varying requirement between Rails 4/5. It's listed in appraisals.